internal/truststore/trust_certificate.go

```diff
@@ -161,6 +161,10 @@ func (r *TrustCertificateResource) ModifyPlan(ctx context.Context, req resource.
 			return
 		}
 	}
+	// Confirm that `certContent` is not an unknown value
```

Comment:
A small nit, but can you also add a new line before the command and a dot at the end of the comment? Just for consistency. Thanks.

Reply:
No worries, have pushed that update :+1: